src/omv/img/mathop.c

```diff
@@ -258,8 +268,8 @@ static void imlib_add_line_op(image_t *img, int line, void *other, void *data, b
                 if ((!mask) || image_get_mask_pixel(mask, i, line)) {
                     int dataPixel = IMAGE_GET_BINARY_PIXEL_FAST(data, i);
                     int otherPixel = IMAGE_GET_BINARY_PIXEL_FAST(((uint32_t *) other), i);
-                    int p = dataPixel + otherPixel;
-                    p = IM_MIN(p, COLOR_BINARY_MAX);
+                    int p = dataPixel | otherPixel; //dataPixel + otherPixel;
```

Comment:
Remove comments.

Reply:
For GRAYSCALE and RGB565 you can use the specialized ARM instructions to go faster.